### ISSUE TITLE
Cannot recognize tap using TalkBack on template

### ISSUE THREAD

**Issue by yama3nomori** [NativeScript/NativeScript#8502]
**Environment**
$ tns info -> Component nativescript has 6.5.0 version and is up to date.
 - Android Runtime: 8.1 / 10

**Describe the bug**
There's a problem on project that be made by tns-template-tab-navigation-ng and tns-template-tab-navigation-vue.
When TalkBack is enabled, double tap on a item cannot be active on home tab.
And also, it is the same problem on tns-template-drawer-navigation-ng and tns-template-drawer-navigation-vue.
There has occured on drawer items.

template by javascript is no problem.
